src/lib.rs

```diff
@@ -59,6 +60,32 @@ fn run_command_with_stdin(command: &mut std::process::Command, input: String) ->
     .unwrap()
     .to_string()
 }
+#[derive(Clone, Copy)]
+pub enum RunType {
+    StructuredConversion,
+    RvsdgConversion,
+    NaiiveOptimization,
```

Comment:
sp: https://en.wiktionary.org/wiki/naive ?

Reply:
hmm idk, I feel like it's spelled Naiive